src/clarity_vm/tests/contracts.rs

```diff
@@ -443,6 +446,72 @@ fn trait_invocation_cross_epoch() {
         publish_contract(conn, &invoke_contract_id, invoke_contract, clarity_version).unwrap();
     });
 
+    info!("Sim height = {}", sim.height);
+    sim.execute_next_block_as_conn(|conn| {
+        let epoch = conn.get_epoch();
+        conn.as_transaction(|clarity_db| {
+            clarity_db
+                .run_contract_call(
+                    &sender,
+                    None,
+                    &invoke_contract_id,
+                    "invocation-1",
+                    &[],
+                    |_, _| false,
+                )
+                .unwrap();
+        });
+    });
+
+    info!("Sim height = {}", sim.height);
+    // now in Stacks 2.2
+    sim.execute_next_block_as_conn(|conn| {
+        let epoch = conn.get_epoch();
+        conn.as_transaction(|clarity_db| {
+            let error = clarity_db
+                .run_contract_call(
+                    &sender,
+                    None,
+                    &invoke_contract_id,
+                    "invocation-2",
```

Comment:
These are the 2.2 blocks. It'd be worthwhile for the unit test to be updated for both of them in the 2.2 blocks. Rather than push the change to this PR, though, I'll open another one, because I don't want to have to run the CI process on this PR again (further delaying release). I have confirmed that it works as expected with the change though.